atomic_reactor/plugins/pre_fetch_sources.py

```diff
@@ -235,8 +236,11 @@ def _get_remote_urls_helper(self, koji_build):
                 dest_name = remote_source['dest']
                 remote_sources_urls.append(remote_source)
 
-            elif archive['type_name'] == 'json' and archive['filename'] == 'remote-source.json':
-                cachito_json_url = os.path.join(remote_sources_path, archive['filename'])
+            elif (
```

Comment:
can you change formatting here? this looks odd

elif (........................... looks nicer